core/src/main/java/jenkins/util/io/CompositeIOException.java

```diff
@@ -32,25 +32,43 @@
 import java.io.UncheckedIOException;
 import java.util.Arrays;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 @Restricted(NoExternalUse.class)
 public class CompositeIOException extends IOException {
-    private final List<IOException> exceptions;
```

Comment:
I wanted to remove this because it is redundant, but I realized that introduces a serialization incompatibility, so I'll revert it. I will also add an explicit `serialVersionUID`.